### ISSUE TITLE
App crashes with modally presented UINavigationViewController in iOS 8

### ISSUE THREAD

**Issue by Jeehut** [problame/CSNotificationView#48]
If I present a UINavigationViewController, show a CSNotificationView view within this modal view controller and dismiss the modal view controller during the CSNotificationView's presentation time, then I receive the following error in iOS 8:

```
Fatal Exception: NSInternalInconsistencyException
An instance 0x[...] of class UINavigationBar was deallocated while key value observers were still registered with it. Current observation info: <NSKeyValueObservationInfo 0x[...]> ( <NSKeyValueObservance 0x[...]: Observer: 0x[...], Key path: bounds, Options: <New: YES, Old: NO, Prior: NO> Context: 0x[...], Property: 0x[...]> )
```

By the way:
The reason why a CSNotificationView might be present on dismiss is that I use CSNotificationView to show Push Notifications when the App is in the foreground. So a CSNotificationView might be added at all times.

**Comment by dustMason** [comment]
Hi, I found this thread after encountering this issue using 0.5.3 in my project. From the comments here I'm not quite sure if that release includes this fix. Does it? Thanks

**Comment by problame** [comment]
See above: [I implemented a fix in 0.5.2](https://github.com/problame/CSNotificationView/commit/a47818a26a96292c2a5ec6e958c477b9807abfc2)
